cli_tools/common/utils/flags/key_value_flag.go

```diff
@@ -0,0 +1,40 @@
+package flags
```

Comment:
It looks like a few of these files need the Apache license header

Reply:
added, thanks!